code/datums/components/omen.dm

```diff
@@ -87,32 +98,63 @@
 					qdel(src)
 				return
 
-/// If we get knocked down, see if we have a really bad slip and bash our head hard
+/** If we get knocked down, see if we have a really bad slip and bash our head hard */
```

Comment:
why the **? js brain?

Reply:
well, yes, but i've been using /** for procs and /// for vars. Says this in the contrib guidelines

Reply:
I didn't know anyone actually read the autodoc page
The general pattern is / ** for longer comments, /// for single lines
Some degens like me stack /// occasionally too, should yell at those people

Reply:
i've been trying to write very brief "why" style comments,, so i'll stick with /// for now